### PR TITLE
Currated paths page - fixes #270

### LINKED ISSUE #270: Currated paths page
Add in menu a link to "Currated paths" page

Display a list of mocked items reflecting paths on that page.

Synopsis:
"If single goals are not enough for you, we will provide you with a complete, curated Path! It will help you achieve one of your goals by guiding you through the steps we have hand crafted specially for you. Want to try Docker? Not sure which course to pick? Look no further, “The Path of the Dockerizer” by the one and only Karol S. will make you use Docker like a champ in no time!"

### PR DESCRIPTION
**Pull request by jacekelgda**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,6 +3,8 @@ dist
 npm-debug.log
 .idea/
 config.js
+.DS_Store
```
Comment (karolsojko) [inline .gitignore]: I think it's better to gitignore DS_Store files on your environment as global gitignore. Here's some details how to do that: http://stackoverflow.com/a/19299889

File: app/config/routes.js [hunk 2]
```diff
@@ -14,5 +15,6 @@ export default (
     <Route path="/skills" component={SkillsContainer} />
     <Route path="/skills/:slug" component={SkillContainer} />
     <Route path="/goals" component={GoalsContainer} />
+    <Route path="/currated_paths" component={CurratedPathsContainer} />
```
Comment (karolsojko) [inline app/config/routes.js]: maybe `/currated-paths` url is better than underscore - what do you think ?

Comment (jacekelgda) [inline app/config/routes.js]: +1

**Comment by jacekelgda** [comment]
@karolsojko updated PR, please check

File: .gitignore (hunk 1 above)
Comment (jacekelgda) [inline .gitignore]: done

File: app/config/routes.js (hunk 2 above)
Comment (jacekelgda) [inline app/config/routes.js]: done

**Pull request merged**